CHANGELOG.asciidoc

```diff
@@ -28,6 +28,21 @@ Notes:
 [[release-notes-3.x]]
 === Node.js Agent version 3.x
 
+==== Unreleased
+
+[float]
+===== Breaking changes
+
+[float]
+===== Features
+
+* Added <<span-frames-min-duration, `spanFramesMinDuration`>>
+  configuration field, allowing users to set a time threshold value that spans
+  must reach before the agent will add a stack trace to the span.
```

Comment:
I think this should be expanded to point out that *by default* now, fast spans won't have stack traces.

Reply:
Fair -- added a _As a result, many short spans that previously included stack traces by default no longer will._